src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -959,6 +955,7 @@ def remove_file_post
           end
 
           it 'shows the complete diff' do
+            skip('Broken by https://github.com/openSUSE/open-build-service/pull/4782')
```

Comment:
I doubt this broke it. It just moved the file limit to a constant. The breakage might have come with the diff truncation, superseeded diffs or one of the other diff related changes.

#4782 "just" didn't add the same kind of stubbing like in the request controller. Seems I missed that back then.

Reply:
just try to regenerate the cassettes for the package_controller_spec to see

Reply:
Already did, and I have a branch which updates this test get it to work again. Just checkout the PR you refer, it does not change any of the code, except storing the threshold in a constant.

The other changes introduced there were to stub that constant and return a lower threshold (= smaller cassettes). The breakage must have happened before or after that and was not noticed because of the cassettes.

I don't mind the reference there. Just wanted to point this out.

Reply:
well, it was just a shot in the dark